Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -517,3 +490,82 @@ open class ChartDataSet: ChartBaseDataSet
         return copy
     }
 }
+
+// MARK: MutableCollection
```

Comment:
if all above benefits come from all the protocols below, should we move the extension to top lines?

Reply:
It's convention to add conformance in extensions.